### PR TITLE
Siftlogic improvements

### PR DESCRIPTION
**Pull request by puzza007**
Thanks, @loucash !

### CODE REVIEW

**Comment by loucash** [comment]
:+1:

File: src/riakc_poolboy.app.src [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {application, riakc_poolboy,
  [
   {description, "Simple Riak PB connection pool"},
-  {vsn, "1.3.1"},
+  {vsn, "1.4.0"},
```
Comment (ibnfirnas) [inline src/riakc_poolboy.app.src]: Wouldn't it be user-friendlier to mirror (at least the major) version of riakc?

This is an actual question - I don't really have a strong opinion :smiley:

Comment (puzza007) [inline src/riakc_poolboy.app.src]: Ah gotcha.  So something like `2.1.1.*`?

Comment (ibnfirnas) [inline src/riakc_poolboy.app.src]: Yeah, something like that. I'm just not sure to also add `riakc_poolboy`-specific increments in a non-ugly way though... `$riakc_version-$riakc_poolboy_version` comes to mind, but seems kinda long...

**Pull request merged**